Dockerfile

```diff
@@ -1,9 +1,15 @@
-FROM scratch
+FROM multiarch/alpine:armhf-v3.4
```

Comment:
This requires `docker run --rm --privileged multiarch/qemu-user-static:register --reset` to be executed on the host building the image.